airbyte-integrations/bases/base-normalization/ssh/sshtunneling.sh

```diff
@@ -0,0 +1,62 @@
+# This function opens an ssh tunnel if required using values provided in config.
+# Requires two arguments,
+  # path to config file ($1)
+  # path to file containing local port to use ($2)
+function openssh() {
+  # check if jq is missing, and if so try to install it..
+  # this is janky but for custom dbt transform we can't be sure jq is installed as using user docker image
+  if ! command -v jq &> /dev/null ; then
+    echo "CRITICAL: jq not installed... attempting to install on the fly but will fail if unable."
+    { apt-get update && apt-get install -y jq; } ||
+    apk --update add jq ||
+    { yum install epel-release -y && yum install jq -y; } ||
+    { dnf install epel-release -y && dnf install jq -y; } || exit 1
+  fi
```

Comment:
I don't like this at all but custom dbt transforms use a user-provided docker image so we can't guarantee libs installed. The install attempts are foremost to pass tests and secondly to try and make things smoother for user.